app/views/layouts/_proxy_notice.html.erb

```diff
@@ -0,0 +1,22 @@
+<div id="proxy-notice">
+  <div>
+    <% # TODO: When the interface is localized, this should be revised to only show the notice in the user's selected language. %>
+    <p><%= t(".text_html") %></p>
+    <p>
+      <% # Doing it as a block saves us from needing to specify the scope. %>
+      <% I18n.with_locale("zh-CN") do %>
```

Comment:
```
<p><%= t(".text_html", locale: :"zh-CN") %></p>
```
This works for me. You may have to [restart Rails](https://stackoverflow.com/questions/20555995/automatically-reload-rails-yml-files-in-config-locales) after making changes to the locale yml files.

Reply:
Yeah, sorry, I did get it working after the third restart -- I didn't realize it hadn't dismissed the comment saying it wasn't working.

Reply:
Oh, do you mean it works for you without specifying the scope? 

I had it working with `<%= I18n.t :text_html, scope: [:layouts, :proxy_notice], locale: :"zh-CN" %>`, then tried switching back to `<%= t(".text_html", locale: :"zh-CN") %>`, but no luck. I didn't make any changes to the locale file during that process -- only to this file -- so I don't think that was the issue.

Reply:
I have no idea if it's working or not, but I've pushed your version. (It _seems_ to be working, but I don't understand why, especially since it was inexplicably insistent about it needing `html_safe` to handle the line breaks this morning despite using `_html` in the key, and now it's quite happy without that. My best guess at this point is browser caching.)

Reply:
It might actually be browser caching. You should check the Cache-Control header on responses from the Rails dev server, and see if it's [enough to get Safari to stop caching](https://stackoverflow.com/a/2068407).